Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -280,7 +294,7 @@ static void CollectAccessibilityElements(ASDisplayNode *node, NSMutableArray *el
     } else if (subnode.isLayerBacked) {
       // Go down the hierarchy of the layer backed subnode and collect all of the UIAccessibilityElement
       CollectUIAccessibilityElementsForNode(subnode, node, view, elements);
-    } else if (subnode.accessibilityElementCount > 0) {
+    } else if (subnode.accessibilityElements.count > 0) {
```

Comment:
Does anyone have any thoughts around this change? When I made an example app with `accessibilityViewIsModal` set to `YES` `accessibilityElementCount` was 1 when I turned on voice over. However, in the tests it was always 0. 

I'm not really sure where/how `accessibilityElementCount` is being changed in my test app. It isn't a property on `NSObject`, but a method that should be overwritten (which we aren't doing). This comment also exists in the header of `_ASDisplayViewAccessiblity.h`, so I'm not sure why we are favoring `accessibilityElementCount` over `accessibilityElements`:

```
// WARNING: When dealing with accessibility elements, please use the `accessibilityElements`
// property instead of the older methods e.g. `accessibilityElementCount()`. While the older methods
// should still work as long as accessibility is enabled, this framework provides no guarantees on
// their correctness.
```

Reply:
as far as I understand we should rely on `accessibilityElementCount`, but
according to documentation it returns `0` by default. May be default implementation? 
```
- (NSInteger)accessibilityElementCount {
  return [[self accessibleElements] count];
}
```

Reply:
I just played with this some more, and it doesn't work. I'll focus on a way to make the tests pass and revert this change.